src/app/core/interceptors/global-http-interceptor.service.ts

```diff
@@ -10,6 +10,7 @@ export class GlobalHttpInterceptorService implements HttpInterceptor {
 intercept(request:HttpRequest<any>,next: HttpHandler): Observable<HttpEvent<any>>{
       return next.handle(request)
       .pipe(
+      //  retry(1),
```

Comment:
you added the line and commented out? then why have you imported retry? please remove if you dont want to use it.